src/shared-component/navbar/index.jsx

```diff
@@ -0,0 +1,158 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import Menu from "@mui/material/Menu";
+import MenuIcon from "@mui/icons-material/Menu";
+import Container from "@mui/material/Container";
+import Button from "@mui/material/Button";
+import MenuItem from "@mui/material/MenuItem";
+import AdbIcon from "@mui/icons-material/Adb";
+import { useNavigate } from "react-router-dom";
+
+const pages = [
+  {
+    title: "Recipe Search",
+    url: "/",
+  },
+  {
+    title: "Create New Recipes",
+    url: "/recipes/new",
+  },
+  {
+    title: "Categories",
+    url: "/recipes/categories",
+  },
+  {
+    title: "Log In",
+    url: "/login",
+  },
+  {
+    title: "Log Out",
+    url: "/logout",
+  },
+];
+
+export const NavBar = () => {
+  const [anchorElNav, setAnchorElNav] = React.useState(null);
+  const navigate = useNavigate();
+
+  const handleOpenNavMenu = (event) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  const handlePageClick = (pageUrl) => {
+    navigate(pageUrl);
+    handleCloseNavMenu();
+  };
+
+  return (
+    <AppBar position="static" color="primary">
+      <Container maxWidth="xl">
```

Comment:
This will make your navbar to stretch further to the sides as you wanted

```suggestion
      <Container maxWidth="xxl">
```